cn_indian_payroll/cn_indian_payroll/overrides/salary_slip.py

```
@@ -1829,14 +1814,15 @@ def calculate_grosspay(self):
         if self.total_loan_repayment:
             self.custom_loan_amount=self.total_loan_repayment
 
-
+#-----------Set tax applicable regime and check boxes from salary component--------------
     def set_taxale(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `set_taxale`

Add return type annotation: `None`

(ANN201)